src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -2,6 +2,7 @@
 
 """
 import pytest
+from numpy import isclose
```

Comment:
Just a comment: I tend to prefer using [`numpy.testing.assert_allclose()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) for all tests involving floats but I know that we've been using `assert isclose()` elsewhere so I am ok with keeping it in your PR.

Reply:
I didn't know about  ```numpy.testing.assert_allclose()```, and now I'm a huge fan as well, I switched to it in the tests!